templates/zerver/help/set-default-streams-for-new-users.md

```diff
@@ -0,0 +1,23 @@
+# Set default streams for new users
+
+Default streams for an organisation is set by an administrator.
+
+Follow the step given below to set the default streams for new users using an administrator account:
```

Comment:
Please make sure your lines are no longer than 76 characters - makes them easier to read in the terminal.